src/Modules/UserProfileModule/components/ChangePassword/ChangePassword.tsx

```diff
@@ -0,0 +1,181 @@
+import { ErrorMessage } from "@hookform/error-message";
+import axios from "axios";
+import { useState } from "react";
+import { useForm } from "react-hook-form";
+import { useNavigate } from "react-router-dom";
+import { toast } from "react-toastify";
+import { passwordValidation } from "../../../../utils/InputsValidation";
+import { baseUrl, getRequestHeaders, loader } from "../../../../utils/Utils";
+import { useUser } from "../../../../Context/AuthContext";
+
+export default function ChangePassword() {
+  const { logout } = useUser();
+  const navigate = useNavigate();
+  // image input state
+
+  // hide pass or show state
+  let [hidePassInInpt, setHidePassInInpt] = useState(true);
+  // check loading state
+  const [isLoading, setIsLoading] = useState(false);
+  // function for change password input type
+  const changePassInputType = () => {
+    setHidePassInInpt((hidePassInInpt = !hidePassInInpt ? true : false));
```

Comment:
We can simplify this as follows:
```
setHidePassInInpt((prev)=>!prev);
```
here we are getting the previous value and negating it.